### PR TITLE
shnsplit: add page; shntool-split: add alias page

### PR DESCRIPTION
**Pull request by Narfinger**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/shnsplit.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# shnsplit
+
+> Splits audio files according to a .cue file.
+> More information: <http://shnutils.freeshell.org/shntool/>.
+
+- Split a wav + cue file into multiple files:
+
+`shnsplit -f file.cue file.wav`
```
Comment (gutjuri) [inline pages/linux/shnsplit.md:8]:
```suggestion
`shnsplit -f {{path/to/file.cue}} {{path/to/file.wav}}`
```

File: pages/linux/shnsplit.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Show supported formats:
+
+`shnsplit -a`
+
+- Split a flac file into multiple files:
+
+`shnsplit -f file.cue -o flac file.flac`
```
Comment (gutjuri) [inline pages/linux/shnsplit.md:16]:
```suggestion
`shnsplit -f {{path/to/file.cue}} -o flac {{path/to/file.flac}}`
```

File: pages/linux/shnsplit.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- Split a wav file into files of the form track-number - album - title:
+
+`shnsplit -f file.cue file.wav -t "%n - %a - %t`
```
Comment (gutjuri) [inline pages/linux/shnsplit.md:20]:
```suggestion
`shnsplit -f {{path/to/file.cue}} {{path/to/file.wav}} -t "%n - %a - %t`
```

File: pages/linux/shnsplit.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# shnsplit
+
+> Splits audio files according to a .cue file.
+> More information: <http://shnutils.freeshell.org/shntool/>.
+
+- Split a wav + cue file into multiple files:
```
Comment (gutjuri) [inline pages/linux/shnsplit.md:6]:
```suggestion
- Split a .wav + .cue file into multiple files:
```

File: pages/linux/shnsplit.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+`shnsplit -f file.cue file.wav`
+
+- Show supported formats:
+
+`shnsplit -a`
+
+- Split a flac file into multiple files:
```
Comment (gutjuri) [inline pages/linux/shnsplit.md:14]:
```suggestion
- Split a .flac file into multiple files:
```

File: pages/linux/shnsplit.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+`shnsplit -f file.cue -o flac file.flac`
+
+- Split a wav file into files of the form track-number - album - title:
```
Comment (gutjuri) [inline pages/linux/shnsplit.md:18]:
```suggestion
- Split a .wav file into files of the form track-number - album - title:
```

**Review by gutjuri — CHANGES_REQUESTED** [review]

**Review by gutjuri — APPROVED** [review]

File: pages/linux/shnsplit.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+- Show supported formats:
+
+`shnsplit -a`
+
+- Split a .flac file into multiple files:
+
+`shnsplit -f {{path/to/file.cue}} -o flac {{path/to/file.flac}}`
+
+- Split a .wav file into files of the form track-number - album - title:
```
Comment (kbdharun) [inline pages/linux/shnsplit.md:18]:
```suggestion
- Split a `.wav` file into files of the form "track-number - album - title":
```

File: pages/linux/shnsplit.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+- Split a .wav + .cue file into multiple files:
+
+`shnsplit -f {{path/to/file.cue}} {{path/to/file.wav}}`
+
+- Show supported formats:
+
+`shnsplit -a`
+
+- Split a .flac file into multiple files:
```
Comment (kbdharun) [inline pages/linux/shnsplit.md:14]:
```suggestion
- Split a `.flac` file into multiple files:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

Seems like `shtool split` is an alias of this so I will add an alias page too here for this.

Ref. 
```zsh
➜  ~ shnsplit
shnsplit: error: if file to be split is not given, then a split point file must be specified
shnsplit: 
shnsplit: type 'shnsplit -h' for help
➜  ~ shntool split
shntool [split]: error: if file to be split is not given, then a split point file must be specified
shntool [split]: 
shntool [split]: type 'shntool split -h' for help
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**